dev/com.ibm.ws.jpa_spec10_injection_fat/test-applications/injection/ejb/injectionDFINoInheritance/META-INF/permissions.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- 
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```

Comment:
Copyright should include 2 years.... 1st year delivered, and last year modified (unless same as 1st year).... so this should be :
```
Copyright (c) 2019, 2020 IBM Corporation and others.
```

Probably true for most of the files where you've changed the copyright.